src/hooks/engine.rs

```diff
@@ -1939,6 +1997,19 @@ pub mod exported {
         })
     }
 
+    #[export_name = "R_LoadSkys"]
+    pub unsafe extern "C" fn my_R_LoadSkys() {
+        abort_on_panic(move || {
+            let marker = MainThreadMarker::new();
+
+            skybox_change::change_name(marker);
```

Comment:
A with_changed_name() function would be good here, see some other with_ in some other hooked function

Reply:
I'm curious about this design. What happens if we have another functionality involving something like that? Wouldn't we have to undo this? I won't try to think too much about it

Reply:
Oh, never mind, Im stupid, it makes sense.